Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6613,20 +6620,26 @@ def prettify_user_interface_config(interface_config: List) -> List:
 def show_user_id_interface_config_request(args):
     template = args.get('template') if args.get('template') else TEMPLATE
     template_stack = args.get('template_stack')
-    vsys = demisto.params().get('vsys')
+    vsys = demisto.args().get('vsys')
```

Comment:
isn't `demisto.args()` the same as `args`?

Reply:
yes, my bad